crates/turbopack-ecmascript/src/transform/mod.rs

```diff
@@ -182,8 +182,13 @@ impl EcmascriptInputTransform {
                 *program = module_program.fold_with(&mut next_ssg(eliminated_packages));
             }
             EcmascriptInputTransform::NextJsFont => {
+                #[allow(unused_mut)] // This is mutated when next-font-local is enabled
+                let mut font_loaders = vec!["@next/font/google".into()];
+                #[cfg(feature = "next-font-local")]
+                font_loaders.push("@next/font/local".into());
```

Comment:
This should happen in `next-core`. You should pass the font loader vec within the`EcmascriptInputTransform::NextJsFont` variant.